azure-pipelines.yml

```diff
@@ -31,6 +31,55 @@ variables:
   backwardCompatibleTestOptions_Linux_3_1: ""
   forwardCompatibleTestOptions_Linux_3_1: ""
 
+  # Skip all backwardCompatible UDF tests since there is a contract change in Spark 3.2 worker.py that affects
+  # UDFs
+  backwardCompatibleTestOptions_Windows_3_2: "--filter \
```

Comment:
A bit curious about backward/forward compatibility testing.  Do we even need to enable or test this when Micorosft.Spark 2.0.0 doesn't have any support for Spark 3.2.x ?  Officially it doesn't support it, so there should be no expectations that there should be backward/forward compat right ?

Reply:
Makes sense, disabled all Spark 3.2 tests